docs/tutorial/05-non-fungible-tokens-2.md

```diff
@@ -13,781 +13,474 @@ meta:
-    access(all) fun mintNFT(): @NFT {
+:::
 
-        // create a new NFT
-        var newNFT <- create NFT(initID: self.idCount)
+Dictionary definitions don't usually have the `@` symbol in the type specification, but because the `myNFTs` mapping stores resources, the whole field must become a resource type.  Therefore, you need the `@` symbol indicating that `ownedNFTs` is a resource type.
 
-        // change the id so that each ID is unique
-        self.idCount = self.idCount + 1
+As a result, all the rules that apply to resources apply to this type.
 
-        return <-newNFT
-    }
+### Utility Functions
 
-	init() {
-        self.CollectionStoragePath = /storage/nftTutorialCollection
-        self.CollectionPublicPath = /public/nftTutorialCollection
-        self.MinterStoragePath = /storage/nftTutorialMinter
+It's helpful for a collection to be able to handle some basic operations, such as accepting an NFT into the collection, validating whether or not a token is present, or sharing a list of all token IDs.
 
-        // initialize the ID count to one
-        self.idCount = 1
+:::info[Action]
 
-        // store an empty NFT Collection in account storage
-        self.account.storage.save(<-self.createEmptyCollection(), to: self.CollectionStoragePath)
+Write a function to `deposit` a token into `ownedNFTs`:
 
-        // publish a capability to the Collection in storage
-        let cap = self.account.capabilities.storage.issue<&Collection>(self.CollectionStoragePath)
-        self.account.capabilities.publish(cap, at: self.CollectionPublicPath)
-	}
+:::
+
+```cadence
+access(all) fun deposit(token: @NFT) {
+    self.ownedNFTs[token.id] <-! token
 }
 ```
 
-This smart contract more closely resembles a contract
-that a project would actually use in production, but still does not use the official NFT standard,
-so it should not be used in any production code.
-
-Any user who owns one or more `ExampleNFT` should have an instance
-of this `@ExampleNFT.Collection` resource stored in their account.
-This collection stores all of their NFTs in a dictionary that maps integer IDs to `@NFT`s.
-
-Each collection has a `deposit` and `withdraw` function.
-These functions allow users to follow the pattern of moving tokens in and out of
-their collections through a standard set of functions.
-
-When a user wants to store NFTs in their account,
-they will create an empty `Collection` by calling the `createEmptyCollection()` function in the `ExampleNFT` smart contract.
-This returns an empty `Collection` object that they can store in their account storage.
-
-There are a few new features that we use in this example, so let's walk through them.
+:::tip
 
-## The Resource Dictionary
+Notice that we're using the `<-!` force assignment operator to move the token.  This will give still give a runtime error if the location is **not** `nil`, but it won't give a typecheck error like the `<-` move operator would in this instance.
```

Comment:
```suggestion
Notice that we're using the `<-!` force assignment operator to move the token.  This will still give a runtime error if the location already has something else stored, but it won't give a typecheck error like the `<-` move operator would in this instance.
```